utils/setup_devcontainer.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#
+# Sets up enviroment for dev in local devcontainer.
+set -e
+
+# Skip when called in a github action workflow
+if [[ !$CI == "true" ]]; then
+    # Add id_rsa so that we can push to github from the dev container
+    # ssh-add $HOME/.ssh/id_rsa
+    ssh-add
```

Comment:
nice

Reply:
I went with the ssh-add method that's working for you, gonna see if it works for me and clean this up later